backend/pkg/db/tables/instances/helper.go

```diff
@@ -0,0 +1,5 @@
+package db_instances
+
+func getInstanceId(usersub string, containerUUID string) string {
```

Comment:
Be consistent with variable names. E.g. "usersub" in helper.go and helper_test.go